src/views/preview/preview.jsx

```diff
@@ -339,6 +356,7 @@ class Preview extends React.Component {
                     className="gui"
                     projectHost={this.props.projectHost}
                     projectId={this.state.projectId}
+                    onClickLogout={this.handleLogout}
```

Comment:
Does this exist yet?

Reply:
Yes, it is earlier in the file.

Reply:
Sorry I mean the `onClickLogout` prop on the `GUI` component. Adding this would probably end up passing it down to the final output component and cause a warning.  It's probably fine for this to get merged out of sync though.